### PR TITLE
SOF-2223 Update ports to call AlbaTv2Server

### PR DESCRIPTION
**Pull request by LindvedKrvang**
Updated Ports so that we know call `AlbaTv2Server` instead of `AlbaServer`.

This changes does require AlbaTv2Server to be running to make the system works.

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -7,9 +7,9 @@ import { LogLevel } from '../app/core/abstractions/logger.service'
 
 export const environment: Environment = {
   production: false,
-  apiBaseUrl: 'http://localhost:3005/api',
+  apiBaseUrl: 'http://localhost:30010/api',
```
Comment (MadsThy) [inline src/environments/environment.ts]:
```suggestion
  apiBaseUrl: 'http://localhost:3010/api',
```

**Review by MadsThy — APPROVED** [review]

**Review by RasmusAlbrektsen — APPROVED** [review]

**Review by KvelaGorrrrnio — APPROVED** [review]
Same as what Mads noticed :)

File: src/environments/environment.ts (hunk 1 above)
Comment (LindvedKrvang) [reply src/environments/environment.ts]: image

Comment (LindvedKrvang) [reply src/environments/environment.ts]: It's been updated

**Pull request merged**